docs/src/__examples__/Heading/DEFAULT.tsx

```diff
@@ -2,7 +2,11 @@ import React from "react";
 import { Heading } from "@kiwicom/orbit-components";
 
 export default {
-  Example: () => <Heading>Orbit components</Heading>,
+  Example: () => (
```

Comment:
What do you think about adding `role` and `level` attrs also for [Typography](https://dsil-add-a11y-heading.orbit.pages.dev/foundation/typography/) section in Orbit docs? Does it make sense for you, although it's not sooo "important"?

Reply:
I don't think it makes a difference, they're not visible. The Typography section should be more to display the different visual variations we have for text